functions/handler.py

```
@@ -653,7 +654,39 @@ def _policy_cas(decision, event: dict) -> dict:
+            session_id=event.get("session_id"),
+            event_id=event.get("event_id"),
+            request_id=event.get("request_id"),
+            trigger_reason=routing.trigger_reason,
+            severity=routing.severity,
+            identity_assertion_text=governance.get("identity_assertion_text"),
+            cas_score=cas.get("score"),
+            cas_tier=cas.get("tier"),
+        )
+        queue_id = row.get("id")
+    except Exception:  # noqa: BLE001 — read-only FS or no DB: still report the routing decision
+        pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)